hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/recon/TestReconTasks.java

```diff
@@ -147,6 +153,12 @@ public void testMissingContainerDownNode() throws Exception {
         scmContainerManager
             .allocateContainer(RatisReplicationConfig.getInstance(ONE), "test");
     long containerID = containerInfo.getContainerID();
+
+    RDBBatchOperation rdbBatchOperation = new RDBBatchOperation();
+    reconContainerMetadataManager
+        .batchStoreContainerKeyCounts(rdbBatchOperation, containerID, 2L);
+    reconContainerMetadataManager.commitBatchOperation(rdbBatchOperation);
```

Comment:
```suggestion
    try (RDBBatchOperation rdbBatchOperation = new RDBBatchOperation()) {
      reconContainerMetadataManager
          .batchStoreContainerKeyCounts(rdbBatchOperation, containerID, 2L);
      reconContainerMetadataManager.commitBatchOperation(rdbBatchOperation);
    }
```

Reply:
@adoroszlai Ah!!, Thanks for pointing out.